src/lbaf/Execution/lbsClusteringTransferStrategy.py

```diff
@@ -40,8 +43,9 @@ def __cluster_objects(self, rank):
             k: clusters[k]
             for k in random.sample(clusters.keys(), len(clusters))}
 
-    def __find_suitable_subclusters(self, clusters, rank_load, r_tol=0.05):
```

Comment:
Will it be a problem that I removed this parameter? From what I could tell, it is never explicitly set (ie only the default value is used). I could of course revert back to how it was, or even set it to the new `cluster_swap_rtol`

Reply:
`r_tol` is the value that should be passed by the configuration parameter. Whether you want to pass it, once retrieved, to this function as is currently done; or rather store it as member variable of the class, is an implementation detail I will let you decide based on what you think is most convenient.

Reply:
Resolved